packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -473,10 +500,13 @@ class InspectorController extends DisposableController
 
   void refreshSelection(RemoteDiagnosticsNode? newSelection) {
     newSelection ??= selectedDiagnostic;
-    setSelectedNode(findMatchingInspectorTreeNode(newSelection));
-    syncSelectionHelper(selection: newSelection);
+    final matchingNode = findMatchingInspectorTreeNode(newSelection);
+    if (matchingNode != null) {
+      setSelectedNode(findMatchingInspectorTreeNode(newSelection));
```

Comment:
Can this just be `setSelectedNode(matchingNode)` to avoid a second call to `findMatchingInspectorTreeNode`?

Reply:
Oops oversight! Done